hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/parse/SqlCreateMappingTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2021 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.sql.impl.parse;
+
+import com.google.common.collect.ImmutableMap;
+import com.hazelcast.sql.impl.schema.Mapping;
+import com.hazelcast.sql.impl.schema.MappingField;
+import com.hazelcast.sql.impl.type.QueryDataType;
+import org.junit.Test;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SqlCreateMappingTest {
```

Comment:
Maybe it would be better if the line endings didn't depend on which OS the cluster runs. But I couldn't find an option in calcite's SqlWriter to configure this, it always uses the system separator. Anyway, it's not a big issue I think.